.github/workflows/continuous-integration-snifftest.yml

```diff
@@ -0,0 +1,48 @@
+name: basic-test
+
+on:
+  push:
+    branches:
+      - develop
+  workflow_call:
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}-cibasic
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+permissions: read-all
+
+jobs:
+  gcc-serial-cpp17: 
+    name: gcc-openmp-cpp17
+    runs-on: [ubuntu-latest]
+
+    steps:
+      - name: checkout code
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```

Comment:
```suggestion
      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
```
You don't really need the `name` to get a meaningful step summary:
![image](https://github.com/user-attachments/assets/f4d67e45-2bfe-4e60-ab4f-aed87f3f4ffa)

Reply:
I usually use `name` when it is not straightforward to grasp what the step does. Here, it is sufficiently clear so you could skip the `name`.